clients/src/test/java/org/apache/kafka/common/serialization/SerializationTest.java

```diff
@@ -119,6 +120,49 @@ public void testLongSerializer() {
     }
 
     @Test
+    public void testFloatSerializer() {
+        Float[] floats = new Float[]{
+            5678567.12312f,
+            -5678567.12341f
+        };
+
+        Serializer<Float> serializer = Serdes.Float().serializer();
+        Deserializer<Float> deserializer = Serdes.Float().deserializer();
+
+        for (Float value : floats) {
+            assertEquals("Should get the original double after serialization and deserialization",
```

Comment:
i'd prefer we use asserThat, i.e., `assertThat(value, equalTo(deserializer.deserialize(...))`